accumulator/src/full.rs

```diff
@@ -485,7 +428,7 @@ mod tests {
         let leaf = H256::repeat_byte(1);
 
         let mut full = MerkleTree::create(&[], TREE_DEPTH);
-        let mut incr = incremental::IncrementalMerkle::default();
+        let mut incr = light::IncrementalMerkle::<32>::default();
```

Comment:
Use `IncrementalMerkle<TREE_DEPTH>` and `Tree<TREE_DEPTH>` instead of hardcoded 32 everywhere? Probably just a search for "<32>" to find occurrences